lib/galaxy/dependencies/dev-requirements.txt

```diff
@@ -89,6 +89,7 @@ markupsafe==1.1.1; (python_version >= "2.7" and python_full_version < "3.0.0") o
 mercurial==5.6.1
 mirakuru==2.3.0; python_version >= "3.6"
 mistune==0.8.4; python_version >= "3.6" and python_version < "4"
+mrcfile==1.2.0; python_version >= "3.6"
```

Comment:
This probably doesn't belong in dev dependencies right?

I think it will need to be in packages/data/requirements.txt .

Reply:
Stumbled over this one too, but poetry writes out full requirements and we don't install both requirements.txt and dev-requirements.txt

Reply:
Just so I'm sure, I should:
- remove the mrcfille entry from lib/galaxy/dependencies/dev-requirements.txt
- keep the current entry in lib/galaxy/dependencies/pinned-requirements.txt 
- add the entry into packages/data/requirements.txt
Thanks for the help!

Reply:
Only the third option, and you should add the requirement to https://github.com/galaxyproject/galaxy/blob/dev/pyproject.toml as well

Reply:
Thanks a bunch @mvdbeek - I believe I've got the correct locations now.